packages/zelus-gtk/zelus-gtk.2.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+synopsis: "A synchronous language with ODEs"
+authors: ["Marc Pouzet <marc.pouzet@ens.fr>"]
+maintainer: ["Marc Pouzet <marc.pouzet@ens.fr>"]
+homepage: "http://zelus.di.ens.fr"
+doc: "http://zelus.di.ens.fr/man/"
+bug-reports: "https://github.com/INRIA/zelus/issues"
+depends: [
+  "ocaml" {>= "4.08.1"}
+  "dune" {>= "2.6"}
+  "zelus" {= version}
+  "lablgtk"
+  "conf-gtksourceview"
```

Comment:
gtksourceview-2.0 is getting really old and is not supported by an increasing number of distributions, it might be more future-proof to switch to gtksourceview3 in next release